src/Components/3DV/SceneView.tsx

```diff
@@ -1663,9 +1681,30 @@ function SceneView(props: ISceneViewProps, ref) {
                     barHeight={10}
                 />
             )}
-            {isLoading === undefined && (
-                <div className={customStyles.errorMessage}>
-                    Error loading model. Try Ctrl-F5
+            {loadingError === 'common' && (
+                <div className={customStyles.commonErrorMessage}>
+                    {t(
+                        'scenePageErrorHandling.3dAssetLoadingCommonErrorMessage'
+                    )}
+                </div>
+            )}
+            {loadingError === 'network' && (
+                <div className={customStyles.networkErrorMessage}>
```

Comment:
can we pull these errors out to a separate component and pass in teh type? this file is already massive. Then it could have common styles for both messages and would simply swap out the text that gets shown/image.